tasks/Receptionist.tex

```diff
@@ -73,26 +83,31 @@ \subsubsection*{To Referee}
 
 The referees need to:
 \begin{itemize}
-	\item Assign name and drink to 3 volunteers.
+	\item Provide the list of names and drinks on setup day
+	\item Assign name, drink and one interest to 3 volunteers.
 	\item Arrange (and re-arrange) people in the living room.
+	\item change the selection in the beverage area
 	\item Open the door when requested by the robot.
 \end{itemize}
 
 \subsubsection*{To OC}
 
+
 During setup day:
 \begin{itemize}
-	\item Provide the dorbell sound (if not using door knocking).
+	\item announce beverage location
+	\item the list of names and drinks is given (a list of interests will not be provided)
```

Comment:
Names is covered under rule:scenario_names, drinks are a defined category.
You can remove this.
```suggestion
	
```